### PR TITLE
Is win

### PR DESCRIPTION
**Pull request by chayahaor**

### CODE REVIEW

File: src/test/java/touro/peg/TriangleBoardTest.java [hunk 1]
```diff
@@ -29,4 +29,44 @@ public void testToString() {
...
+        }
+        //Then
+        String expected = """
+                    1
+                   0 0
+                  0 0 0
+                 0 0 0 0
+                0 0 0 0 0""";
+        assertEquals(expected, board.toString());
+    }
```
Comment (andrewoid) [inline src/test/java/touro/peg/TriangleBoardTest.java]: If you are going to write test methods then they need to be testing the methods that you're writing.

Neither of these tests call isWin() or isBestWin()

File: src/main/java/touro/peg/TriangleBoard.java [hunk 2]
```diff
@@ -36,4 +38,17 @@ public String toString() {
...
     }
+
+    public boolean isWin(){
+        int sum = 0;
+        for (int i = 0; i < pegs.length; i++)
+        {
+            sum+=intPeg(i);
+        }
+        return sum == 1;
+    }
```
Comment (andrewoid) [inline src/main/java/touro/peg/TriangleBoard.java]: try to simplify this

**Review by andrewoid — APPROVED** [review]

File: src/test/java/touro/peg/TriangleBoardTest.java [hunk 3]
```diff
@@ -29,4 +29,33 @@ public void testToString() {
...
+
+    @Test
+    public void isWin(){
+        //Given
+        //When
+        TriangleBoard board = new TriangleBoard(0);
+        for (int i = 2; i < 15; i++)
+        {
+            board.setPeg(i, false);
+        }
```
Comment (andrewoid) [inline src/test/java/touro/peg/TriangleBoardTest.java:40]: This should be in the given.

File: src/test/java/touro/peg/TriangleBoardTest.java [hunk 4]
```diff
@@ -29,4 +29,33 @@ public void testToString() {
...
+    @Test
+    public void isBestWin(){
+        //Given
+        //When
+        TriangleBoard board = new TriangleBoard(0);
+        board.setPeg(0, true);
+        for (int i = 1; i < 15; i++)
+        {
+            board.setPeg(i, false);
+        }
```
Comment (andrewoid) [inline src/test/java/touro/peg/TriangleBoardTest.java:54]: This should be in the given.

**Comment by andrewoid** [comment]
You should test isWin() false and isBestWin() false.

File: src/main/java/touro/peg/TriangleBoard.java [hunk 5]
```diff
@@ -36,4 +41,21 @@ public String toString() {
...
+        int numTrues = 0;
+
+        for (boolean peg : pegs)
+        {
+            if (peg) {
+                numTrues++;
+            }
+        }
+        return numTrues == 1;
+    }
```
Comment (andrewoid) [inline src/main/java/touro/peg/TriangleBoard.java]: Do this without "numTrues"

**Review by andrewoid — CHANGES_REQUESTED** [review]

File: src/main/java/touro/peg/TriangleBoard.java (hunk 5 above)
Comment (chayahaor) [reply src/main/java/touro/peg/TriangleBoard.java]: How would I do that? Does it require a library or is it a built-in Java feature?

**Review by andrewoid — APPROVED** [review]

**Pull request merged**